src/debug-tools/components/telemetry-viewer/telemetry-messages-list.tsx

```diff
@@ -91,4 +91,4 @@ export const onRenderTimestamp = (
 
 export const defaultDateFormatter: DateFormatter = timestamp =>
     // untested line: moment is sensitive to the host machine time zone
```

Comment:
Dangling reference to moment in comment; is this still true for luxon?

Reply:
Yes, but will update the comment

Reply:
(the format string is also wrong (see: not tested))